src/opentype.js

```diff
@@ -368,6 +368,18 @@ function parseBuffer(buffer, opt) {
     if (gsubTableEntry) {
         const gsubTable = uncompressTable(data, gsubTableEntry);
         font.tables.gsub = gsub.parse(gsubTable.data, gsubTable.offset);
+        font.tables.gsub.features.forEach(f => {
```

Comment:
I stumbled across "uiName" vs. "UiLabelName" at first, but turns out that's in the spec as well, just in case anyone in the future wonders why this is not the same.

Reply:
this library is already pretty slow, I don't personally like using forEach due to the excess function calls compared with a for-in loop, would it be possible to switch this to a for-in loop?

Reply:
done!